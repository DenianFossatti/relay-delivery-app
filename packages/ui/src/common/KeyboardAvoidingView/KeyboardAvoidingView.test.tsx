import { mockTestID, render, wrapper } from '../../tests';

import { KeyboardAvoidingView, KeyboardAvoidingViewProps } from '.';

describe(`KeyboardAvoidingView`, () => {
  const props: KeyboardAvoidingViewProps = {
    testID: 'KeyboardAvoidingView',
  };

  const el_container = mockTestID('View', props.testID!);

  it(`render component #${el_container}`, async () => {
    const { findByTestId, toJSON } = render(<KeyboardAvoidingView {...props} />, { wrapper });
    const sut = await findByTestId(el_container);
    expect(sut).toBeTruthy();
    expect(toJSON()).toMatchSnapshot();
  });
});
