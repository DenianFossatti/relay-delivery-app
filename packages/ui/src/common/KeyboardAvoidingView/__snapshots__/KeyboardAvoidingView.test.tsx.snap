// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/KeyboardAvoidingView render component #View_KeyboardAvoidingView 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    accessible={false}
    onLayout={[Function]}
    style={
      [
        {
          "flex": 1,
          "paddingBottom": 0,
        },
        {
          "paddingBottom": 0,
        },
      ]
    }
    testID="View_KeyboardAvoidingView"
  />
</RNCSafeAreaProvider>
`;
